centralized-sampling-tests/collector-config.yml

```diff
@@ -13,7 +13,7 @@ receivers:
 
 exporters:
   logging:
-    loglevel: debug
```

Comment:
`loglevel` is deprecated. Switch to `verbosity`. https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter